sql/moz-fx-data-shared-prod/telemetry/clients_last_seen_v2/view.sql

```diff
@@ -1,26 +1,140 @@
 CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.telemetry.clients_last_seen_v2`
 AS
+WITH days_since AS (
+  SELECT
+    DATE_DIFF(submission_date, first_seen_date, DAY) AS days_since_first_seen,
+    DATE_DIFF(submission_date, second_seen_date, DAY) AS days_since_second_seen,
+    mozfun.bits28.days_since_seen(days_seen_bits) AS days_since_seen,
+    mozfun.bits28.days_since_seen(days_visited_1_uri_bits) AS days_since_visited_1_uri,
+    mozfun.bits28.days_since_seen(days_visited_5_uri_bits) AS days_since_visited_5_uri,
+    mozfun.bits28.days_since_seen(days_visited_10_uri_bits) AS days_since_visited_10_uri,
+    mozfun.bits28.days_since_seen(days_had_8_active_ticks_bits) AS days_since_had_8_active_ticks,
+    mozfun.bits28.days_since_seen(days_opened_dev_tools_bits) AS days_since_opened_dev_tools,
+    mozfun.bits28.days_since_seen(days_created_profile_bits) AS days_since_created_profile,
+    mozfun.bits28.days_since_seen(days_interacted_bits) AS days_since_interacted,
+    mozfun.bits28.days_since_seen(
+      days_visited_1_uri_bits & days_interacted_bits
+    ) AS days_since_qualified_use_v1,
+    mozfun.bits28.days_since_seen(
+      days_visited_1_uri_normal_mode_bits
+    ) AS days_since_visited_1_uri_normal_mode,
+    mozfun.bits28.days_since_seen(
+      days_visited_1_uri_private_mode_bits
+    ) AS days_since_visited_1_uri_private_mode,
+    *
+  FROM
+    `moz-fx-data-shared-prod.telemetry_derived.clients_last_seen_v2`
+)
 SELECT
-  cls.* EXCEPT (app_name),
+  -- Segment definitions; see https://docs.telemetry.mozilla.org/concepts/segments.html
+  -- 0x0FFFFFFE is a bitmask that accepts the previous 27 days, excluding the current day (rightmost bit)
+  -- 0x183060C183 == 0b000001100000110000011000001100000110000011 is a bit mask that accepts a pair of
+  -- consecutive days each week for six weeks; the current day and previous day are accepted.
+  BIT_COUNT(days_seen_bits & 0x0FFFFFFE) >= 14 AS is_regular_user_v3,
```

Comment:
Looks like all activity fields from here on should align with the definition of active, meaning use days_active_bits instead of days_seen_bits, @bochocki what do you think, shall we implement this is a later PR after that is confirmed?

Reply:
Good catch, @lucia-vargas-a! I was able to confirm yesterday that we should use `days_active_bits` for activity segments and documented the change in the proposal.

Reply:
So we can change these in this PR.